### ISSUE TITLE
npm install -g @angular/cli won't work

### ISSUE THREAD

**Issue by jckuhl** [angular/angular-cli#9790]
Node: v9.4.0
npm: 5.6.0

I cannot get angular to work on my machine.  It seems like every time I try to do anything with the CLI, something goes wrong.  I couldn't get React to work either.  I tried reinstalling NPM, but even with sudo, it kept giving me permission errors.  I'm even thinking about ripping out everything Node and NPM related and reinstalling all of it.  What am I doing wrong?

Jonathans-MBP:~ jonathankuhl$ sudo npm install -g @angular/cli
npm ERR! path /usr/local/bin/ng
npm ERR! code EEXIST
npm ERR! Refusing to delete /usr/local/bin/ng: ../lib/node_modules/angular-cli/bin/ng symlink target is not controlled by npm /usr/local/bin
npm ERR! File exists: /usr/local/bin/ng
npm ERR! Move it away, and try again.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jonathankuhl/.npm/_logs/2018-02-26T22_11_42_128Z-debug.log
Jonathans-MBP:~ jonathankuhl$ ng new ang_app
-bash: ng: command not found
Jonathans-MBP:~ jonathankuhl$

**Comment by scttcper** [comment]
did you happen to install it using homebrew? `brew info angular-cli`

**Comment by jroneil** [comment]
if you do ng new newApp
then do ng serve it works 
then go back to the app you were trying to do ng serve it will work
I also change the cli version in the package.json(I am not sure this is necessary )

**Comment by Enuvid** [comment]
It seems this file already exist. Try to run ng using full path (/usr/local/bin/ng) in terminal. If it works out, just add path '/usr/local/bin/' environment variables and restart terminal.

**Comment by filipesilva** [comment]
This seems like a problem with your node/npm and not with Angular CLI. 

Please have a look at the [fixing npm permissions page](https://docs.npmjs.com/getting-started/fixing-npm-permissions), [common errors page](https://docs.npmjs.com/troubleshooting/common-errors), [npm issue tracker](https://github.com/npm/npm/issues), or open a new issue in that issue tracker if the problem you are experiencing isn't known.

The usage of `sudo` in your report strongly points to permissions issue. I'd advise checking the first link in the paragraph above.

**Comment by jckuhl** [comment]
I used sudo because I'm on a mac, and it seems sudo is necessary for mac users.

Anyways, I did end up fixing this by deleting whatever it was my first attempt installed and trying again.  Maybe it was corrupted or didn't download right for some reason.  However, the ng command is working now.  Thanks.

I still wonder if I might have some sort of problem with NPM though, because I did have issues with the React commands, but that's an issue for another time.
